src/anemoi/training/losses/combined.py

```diff
@@ -0,0 +1,137 @@
+# (C) Copyright 2024 ECMWF.
```

Comment:
@HCookie Can you update the headers to be consistent with anemoi-training? (Sorry I realise this has happened while this has been under review)